src/pulp/solvers.py

```diff
@@ -1420,7 +1420,11 @@ def solve_CBC(self, lp, use_mps=True):
         args.append(self.path)
         args.extend(cmds[1:].split())
         cbc = subprocess.Popen(args, stdout = pipe, stderr = pipe)
-        if cbc.wait() != 0:
+
+        if processTimeout is not None and sys.version_info < (3, 3):
+            warnings.warn('Pulp: Subprocess timeout is not available in python < 3.3. Solver will run indefinitely.')
```

Comment:
The message might be misleading because the solver will not run indefinitely when a timeout is passed to CBC or a solution is found.

Thus I suggest to log `... < 3.3. Passed subprocess timeout is ignored.` instead.